### PR TITLE
Blog app - creating a data model

### PR DESCRIPTION
**Pull request by Doheera-kosi**
# Milestone objectives

- [ ] Install Ruby on Rails framework.
- [ ] Use Ruby Gems as a software packages system.
- [ ] Understand what ORM is.
- [ ] Use database migration files to maintain database schema.

## General requirements

- No linter errors
- Used correct flow [Gitflow]
- Documented work [in a professional way]

## Ruby requirements
Followed our list of [best practices for Ruby]

## Project requirements
- Created a new app. 
- Initialized project with Git.
- Postgres database set up.
- Build project schema.
- Created and run the necessary migration files.
- Table and column names match the ERD diagram.
- Foreign keys are included.
- All columns that are foreign keys have a corresponding index.

### CODE REVIEW

File: config/database.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+#   gem install sqlite3
+#
+#   Ensure the SQLite 3 gem is defined in your Gemfile
+#   gem "sqlite3"
+#
+default: &default
+  adapter: sqlite3
+  pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
+  timeout: 5000
+
```
Comment (emmanuellekamwa) [inline config/database.yml:11]: - [ ] The database adapter is using Postgresql and not sqlite3. Please kindly fix that.

**Review by emmanuellekamwa — APPROVED** [review]
**STATUS: CHANGES REQUESTED ⛔🛑**
Hello @Doheera-kosi 👋
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- You documented your work professionally.✔
- Your linters are passing all the checks.

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: config/database.yml (hunk 1 above)
Comment (Doheera-kosi) [reply config/database.yml:11]: Thanks for the constructive feedback

**Review by alexander16108 — APPROVED** [review]
Hi @Doheera-kosi , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

 

## Highlights

- Good commit history ✔️
- Good PR description ✔️
- Your README file is professional ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**